runtime_scan/pkg/scanner/scanner.go

```diff
@@ -229,13 +231,13 @@ func (s *Scanner) initScan() error {
 			}
 			// Create pod context
 			podContext := &imagePodContext{
-				containerName:   container.Name,
-				podName:         pod.GetName(),
-				namespace:       pod.GetNamespace(),
-				imagePullSecret: k8sutils.GetMatchingSecretName(secrets, container.Image),
-				imageName:       container.Image,
-				podUID:          string(pod.GetUID()),
-				podLabels:       labels.Set(pod.GetLabels()),
+				containerName:    container.Name,
+				podName:          pod.GetName(),
+				namespace:        pod.GetNamespace(),
+				imagePullSecrets: imagePullSecretNames,
```

Comment:
please delete `GetMatchingSecretName` and `GetPodImagePullSecrets` functions if not used now.